.github/actions/python-regression-tests/action.yml

```diff
@@ -37,6 +37,7 @@ runs:
   using: composite
   steps:
     - name: Debug inputs
+      if: ${{ runner.os != 'Windows' }}
```

Comment:
Why all Windows and not only vs19 or vs22?

Reply:
this actions is used by linux and windows runners and there is no difference, which version of VS is used